win32_threads.c

```diff
@@ -685,6 +685,14 @@ static ptr_t copy_ptr_regs(word *regs, const CONTEXT *pcontext) {
 #   elif defined(X86_64)
       PUSH4(Rax,Rcx,Rdx,Rbx); PUSH2(Rbp, Rsi); PUSH1(Rdi);
       PUSH4(R8, R9, R10, R11); PUSH4(R12, R13, R14, R15);
+      PUSH4(Xmm0.Low, Xmm0.High, Xmm1.Low, Xmm1.High);
```

Comment:
I have a question: do we need to save High parts too?

Reply:
https://newbedev.com/128-bit-values-from-xmm-registers-to-general-purpose says SSE4.1 has pextrq to extract High part of xmm but does gcc/clang practically use it?

Reply:
Yes, gcc also uses the high parts. In the example above, the two pointers `a` and `b` are stored in the low and high parts of the register respectively.